### PR TITLE
Bugfix FXIOS-10382 [Focus] Fixed the 'studies' toggle function

### PR DESCRIPTION
**Pull request by Umair-ios**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-10382)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/22752)

## :bulb: Description
To sync 'studies' toggle with 'Send data usage', Added extra check before turning 'studies' toggle on, 

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [x] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Review by MattLichtenstein — CHANGES_REQUESTED** [review]
Looks good, just a little spacing cleanup.

File: focus-ios/Blockzilla/Settings/Controller/SettingsViewController.swift [hunk 1]
```diff
@@ -558,20 +572,22 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
...
+                    disableAndTurnOffStudiesToggle(studiesToggle)
                 } else {
                     // Restore toggle's appearance
                     studiesToggle.isEnabled = true
                     studiesToggle.alpha = 1.0
                 }
             }
         } else if toggle.setting == .studies {
-            NimbusWrapper.shared.nimbus.globalUserParticipation = sender.isOn
+
```
[the remark this answers is not in the document]
Comment (Umair-ios) [reply focus-ios/Blockzilla/Settings/Controller/SettingsViewController.swift]: Removed in [edd23ad](https://github.com/mozilla-mobile/firefox-ios/pull/23724/commits/edd23adefe3c2c83fca30ed05bccfa8e10e78928)

File: focus-ios/Blockzilla/Settings/Controller/SettingsViewController.swift [hunk 2]
```diff
@@ -558,20 +572,22 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
...
+            // Ensure 'studies' is disabled if 'sendAnonymousUsageData' is turned off, even when 'studies' is being enabled.
+            if sendAnonymousUsageDataToggle?.isOn == true {
+                NimbusWrapper.shared.nimbus.globalUserParticipation = sender.isOn
+            } else {
+                disableAndTurnOffStudiesToggle(sender)
+            }
+
```
[the remark this answers is not in the document]
Comment (Umair-ios) [reply focus-ios/Blockzilla/Settings/Controller/SettingsViewController.swift]: Removed in [edd23ad](https://github.com/mozilla-mobile/firefox-ios/pull/23724/commits/edd23adefe3c2c83fca30ed05bccfa8e10e78928)

**Review by MattLichtenstein — APPROVED** [review]
Awesome, thank you @Umair-ios for the contribution 🎉

**Pull request merged**